torch_xla/csrc/ir.cpp

```diff
@@ -101,11 +121,14 @@ XlaNode::XlaNode(torch::lazy::OpKind op, torch::lazy::Shape shape,
     : torch::lazy::Node(op, shape, num_outputs),
       xla_shape_(std::move(xla_shape)),
       node_hash_(GetOpHash(op, xla_shape_, hash_seed)),
-      dag_hash_(node_hash_) {}
+      dag_hash_(node_hash_),
+      oplist_(std::nullopt) {}
```

Comment:
Along the same line with my previous [comment](https://github.com/pytorch/xla/pull/4555/files#r1114930890), we are introducing a gap here between `operands_`, `operands_as_outputs_` and `oplist_` (empty). And we probably want to avoid this additional null initialization if possible.

Reply:
I was planning to close the upstream PR and localize these changes in XlaNode. Do you think we should move it back to lazy::Node like what I had before?